SB2018/2018-02-07.md

```diff
@@ -22,7 +22,7 @@ In this release, we've introduced 2 new coordinate systems, and depricated an ex
 
 As many have noted the [`cartesian`](https://docs.sourcebots.co.uk/api/vision/coordinates/#cartesian-coordinates) system was not implemented, and would raise an exception whenever accessed. This has now been fixed.
  
-The `polar` coordinate system has been replaced with [`spherical`](https://docs.sourcebots.co.uk/api/vision/coordinates/#spherical-coordinates). The new system is more accurate, and better documented than polar, and we recommend updating to it as soon as possible! The available attributes are identical in `polar`, and so will likely be a simple drop-in replacement. Be sure to consult the documentation on exactly now to use the new system. Attempting to use `polar` will still work as it does now, however displays a warning message.
+The `polar` coordinate system has been replaced with [`spherical`](https://docs.sourcebots.co.uk/api/vision/coordinates/#spherical-coordinates). The new system is easier to understand, and better documented than polar. We recommend updating as soon as possible! The available attributes are identical in `polar`, and so will likely be a simple drop-in replacement. Be sure to consult the documentation on exactly now to use the new system. Attempting to use `polar` will still work as it does now, however displays a warning message.
```

Comment:
> however displays a warning message

Nitpick: maybe "prints" would be clearer than "displays" since there isn't a display?